code/visualize.py

```diff
@@ -31,6 +37,14 @@ def format_month_values(month_tag):
     return ''
 
 
-data_dir = 'results/word_count'
-for filename in os.listdir(data_dir):
-    create_single_file_chart(f'{data_dir}/{filename}')
+def main():
+    # visualize_basic_files('results/word_count')
+    visualize_basic_files('results/meta_data_count', True)
+
+
+def visualize_basic_files(data_dir, use_word_from_file=False):
+    for filename in os.listdir(data_dir):
+        create_single_file_chart(f'{data_dir}/{filename}', use_word_from_file)
+
+
+main()
```

Comment:
Add `if __name__=='__main__'`...
This is important to ensure that this code doesn't run when importing functions from this file.
Pycharm has a cool way to generate this. Just type `main` and press `tab`.